packages/strapi-parts/src/BaseRadio/BaseRadio.stories.mdx

```diff
@@ -81,7 +79,7 @@ Description...
 
 ## LSize
 
-Description...
+A bigger size can be used.
```

Comment:
i'm not sure the case works on Storybook. it seems the same size

Reply:
From what I remember, we decided to go for only one size. However, the wording is fine.